front/src/applications/editor/data/utils.ts

```diff
@@ -134,6 +134,7 @@ export function nestEntity(entity: EditorEntity, type: EditoastType): EditorEnti
 
       if (isLast) props[k] = oldProperties[key];
       else props[k] = props[k] || {};
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-return
       return props[k];
```

Comment:
Here we're dealing with a `Record` with `any` values, and we need to pass-through `any` values when merging.